Mathlib/Data/MvPolynomial/CommRing.lean

```diff
@@ -70,6 +70,20 @@ theorem C_neg : (C (-a) : MvPolynomial σ R) = -C a :=
 set_option linter.uppercaseLean3 false in
 #align mv_polynomial.C_neg MvPolynomial.C_neg
 
+variable {σ} in
+lemma X_sub_C_ne_zero (i : σ) (c : R) [Nontrivial R] : X i - C c ≠ 0 := by
+  convert X_add_C_ne_zero i (- c) using 1
+  rw [C_neg, sub_eq_add_neg]
+
+variable {σ} in
+lemma X_sub_C_not_isUnit (i : σ) (c : R) [Nontrivial R] : ¬ IsUnit (X i - C c) := by
```

Comment:
Do we have the `X + C` version too? I think it would make sense to prove this one from that one

Reply:
done, thanks